docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/MulticlassClassification.ttinclude

```diff
@@ -75,7 +75,7 @@ namespace Samples.Dynamic.Trainers.MulticlassClassification
         private static IEnumerable<DataPoint> GenerateRandomDataPoints(int count, int seed=0)
         {
             var random = new Random(seed);
-            float randomFloat() => (float)random.NextDouble();
+            float randomFloat() => (float)(random.NextDouble() - 0.5);
```

Comment:
Why do we do this?

Reply:
@natke Its to make sure feature values are evenly distributed between -0.5 and +0.5. This gives us even number of positive and negative examples. Naive Bayes considers two types of feature values 1) greater than zero and 2) less than equal to zero and you want to have a sample with both those feature values to have sensible prediction. I believe @ganik has talked about it briefly in the doc that he has attached here.

Reply:
Ok, great. Is it worth adding a comment to the code? Also, which doc?

Reply:
@natke I believe he has already [here](https://github.com/dotnet/machinelearning/blob/8831b0f6ba48a001a0b4dcfde896fe5354623f8f/src/Microsoft.ML.StandardTrainers/Standard/MulticlassClassification/MulticlassNaiveBayesTrainer.cs#L30)

Reply:
Ok, so yes it is spelt out in the trainer code comments. I wonder if we should add a comment to this sample code too, to be absolutely clear.

Reply:
I cant add it to the sample code since this code is shared (generated from .tt which is shared) by 3 other trainers that don't have this NaiveBayes problem

---
In reply to: [274708597](https://github.com/dotnet/machinelearning/pull/3246#discussion_r274708597) [](ancestors = 274708597)

Reply:
so we have random values from -.5 to .5 range, some trainers like NB need that, others like OVA dont but will be ok with that 

---
In reply to: [274651105](https://github.com/dotnet/machinelearning/pull/3246#discussion_r274651105) [](ancestors = 274651105)

Reply:
Actually I think I know how to do it, I ll send next iteration

---
In reply to: [275193160](https://github.com/dotnet/machinelearning/pull/3246#discussion_r275193160) [](ancestors = 275193160,274708597)